suites/src/main/java/org/hiero/block/suites/utils/BlockAccessUtils.java

```diff
@@ -0,0 +1,76 @@
+// SPDX-License-Identifier: Apache-2.0
+package org.hiero.block.suites.utils;
+
+import static java.util.Objects.requireNonNull;
+
+import edu.umd.cs.findbugs.annotations.NonNull;
+import org.hiero.block.api.protoc.BlockAccessServiceGrpc;
+import org.hiero.block.api.protoc.BlockRequest;
+import org.hiero.block.api.protoc.BlockResponse;
+
+/**
+ * Utility class for block access operations.
+ * Contains methods for creating block requests and interacting with the Block Node gRPC service.
+ */
+public final class BlockAccessUtils {
+
+    private BlockAccessUtils() {
+        // Prevent instantiation
+    }
+
+    /**
+     * Creates a SingleBlockRequest to retrieve a specific block.
+     *
+     * @param blockNumber The block number to retrieve
+     * @param latest Whether to retrieve the latest block
+     * @return A SingleBlockRequest object
+     */
+    public static BlockRequest createSingleBlockRequest(long blockNumber, boolean latest) {
```

Comment:
Please note renaming on protobuf size and align there
```suggestion
    public static BlockRequest createGetBlockRequest(long blockNumber, boolean latest) {
```

Reply:
Thanks, changed 👍